next.config.mjs

```diff
@@ -1,8 +1,11 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-    compiler: {
-        styledComponents: true,
-    }
+  compiler: {
+    styledComponents: true,
+  },
+  images: {
+    domains: ["drive.google.com", "servicosweb.cnpq.br", "avatars.githubusercontent.com"],
```

Comment:
In next 14 the images.domains is now depracated, we have to use the images.remotePatterns now

```suggestion
  images: {
    remotePatterns: [
      {
        protocol: "https",
        hostname: "drive.google.com",
        port: "",
        pathname: "**",
      },
      {
        protocol: "http",
        hostname: "servicosweb.cnpq.br",
        port: "",
        pathname: "**",
      },
      {
        protocol: "https",
        hostname: "avatars.githubusercontent.com",
        port: "",
        pathname: "**",
      },
    ],
  },
```